youtube_dl/extractor/cammodels.py

```diff
@@ -84,6 +62,14 @@ def _real_extract(self, url):
                         # hls skips fragments, preferring rtmp
                         'preference': -1,
                     })
+                elif format_id == 'jpeg':
+                    thumbnails.append({
+                        'url': f.get('url'),
+                        'width': f.get('width'),
+                        'height': f.get('height'),
+                        'format_id': f.get('format_id')
+                    })
+                    continue
                 else:
```

Comment:
Improve type checking, etc.
```suggestion
                else:
                    if format_id == 'jpeg':
                        t_url = url_or_none(f.get('url'))
                        if t_url:
                            thumbnails.append({
                                'url': t_url,
                                'width': int_or_none(f.get('width')),
                                'height': int_or_none(f.get('height')),
                                'format_id': f.get('format_id')
                            })
```

Reply:
Hey thanks for the review 👍.

I believe these values would have already been validated in lines 38 - 49. Implementing the suggested change would be redoing previously done checking.

Reply:
Oh, sure: but then you can say `f['url']`, etc.